.github/workflows/docs.yml

```
@@ -100,13 +100,25 @@ jobs:
         if: always()
         run: tar -zcvf docs-build.tar.gz doc/_build/html/
 
+      - name: Tar build multiversion
+        if: always()
+        working-directory: ./doc/_build/html/
+        run: tar -zcvf docs-build-multi.tar.gz .
+
       - name: Upload HTML documentation
         if: always()
         uses: actions/upload-artifact@v3
         with:
           name: docs-build
           path: docs-build.tar.gz
 
+      - name: Upload HTML multiversion documentation 
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: docs-build-multi
+          path: docs-build-multi.tar.gz
```

The `working-directory`directive in line 105 only applies for that job step. Hence, for this other step you still need to point to the location of the artifact. Thus:

```suggestion
          path: doc/_build/html/docs-build-multi.tar.gz
```